DataConnectors/WireXsystemsNFP.json

```diff
@@ -0,0 +1,119 @@
+{
+    "id": "WireX_Systems_NFP",
+    "title": "WireX Network Forensics Platform",
+    "publisher": "WireX_Systems",
+    "descriptionMarkdown": "The WireX Systems data connector allows security professional to integrate with Azure Sentinel to allow you to further enrich your forensics investigations; to not only encompass the contextual content offered by WireX but to analyze data from other sources, and to create custom dashboards to give the most complete picture during a forensic investigation and to create custom workflows.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "CommonSecurityLog",
+            "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"WireX Systems\"\n| where DeviceProduct == \"WireX NFP\"\n"
```

Comment:
The DeviceVendor name is WireX only right? If yes, please correct this. Else the data connector will show as not connected and will be a customer issue. If you have data streams that have WireX and WireX Systems a better query would be to do 'contains' as follows:

"baseQuery": "\nCommonSecurityLog\n| where DeviceVendor contains \"WireX\"\n| where DeviceProduct == \"WireX NFP\"\n"